docs/src/index.md

```diff
@@ -0,0 +1,16 @@
+# DataJoint Element - Lab
```

Comment:
Please edit this and the readme to start with a more general description following the structure of element-deeplabcut e.g., 
> DataJoint Element for lab metadata. DataJoint Elements collectively standardize and automate data collection and analysis for neuroscience experiments. Each Element is a modular pipeline for data storage and processing with corresponding database tables that can be combined with other Elements to assemble a fully functional pipeline.

The info below can then turn into a short follow-up sentence